### PR TITLE
CMR-5000

### PR DESCRIPTION
**Pull request by ygliuvt**
Fixed CMR does not check for token expiration during ingest.

### CODE REVIEW

File: mock-echo-app/src/cmr/mock_echo/data/token_db.clj [hunk 1]
```diff
@@ -35,7 +38,11 @@
   :id of the token that was created"
   [context token-info]
   (let [token-db (context->token-db context)
-        token (assoc token-info :id (new-token-id token-db))]
+        token (assoc token-info :id (new-token-id token-db))
+        token (if (:expires token)
+                token
+                ;; token expires in 30 days
+                (assoc token :expires (datetime-helper/datetime->string (t/plus (tk/now) (t/days 30)))))]
```
Comment (daniel-zamora) [inline mock-echo-app/src/cmr/mock_echo/data/token_db.clj]: We should have this be a defconfig

File: transmit-lib/src/cmr/transmit/echo/tokens.clj [hunk 2]
```diff
@@ -47,7 +50,14 @@
                                                        "Echo-Token" (transmit-config/echo-system-token)}
                                              :form-params {:id token}})]
       (case (int status)
-        200 (get-in parsed [:token_info :user_name])
+        200 (let [expires (some-> (get-in parsed [:token_info :expires])
```
Comment (daniel-zamora) [inline transmit-lib/src/cmr/transmit/echo/tokens.clj]: Could this be an if-let?

Comment (daniel-zamora) [reply transmit-lib/src/cmr/transmit/echo/tokens.clj]: maybe not, disregard.

File: mock-echo-app/src/cmr/mock_echo/data/token_db.clj (hunk 1 above)
Comment (ygliuvt) [reply mock-echo-app/src/cmr/mock_echo/data/token_db.clj]: This code is only for testing. We can make this a `def`, but probably not a `defconfig`.

**Pull request merged**